bazel/tools/java/com/code_intelligence/jazzer/tools/FuzzTargetTestWrapper.java

```diff
@@ -46,16 +46,25 @@ public static void main(String[] args) {
     String jarActualPath;
     boolean verifyCrashInput;
     boolean verifyCrashReproducer;
+    boolean expectCrash;
     Set<String> expectedFindings;
+    List<String> arguments;
     try {
       runfiles = Runfiles.create();
       driverActualPath = lookUpRunfile(runfiles, args[0]);
       apiActualPath = lookUpRunfile(runfiles, args[1]);
       jarActualPath = lookUpRunfile(runfiles, args[2]);
       verifyCrashInput = Boolean.parseBoolean(args[3]);
       verifyCrashReproducer = Boolean.parseBoolean(args[4]);
+      expectCrash = Boolean.parseBoolean(args[5]);
       expectedFindings =
-          Arrays.stream(args[5].split(",")).filter(s -> !s.isEmpty()).collect(Collectors.toSet());
+          Arrays.stream(args[6].split(",")).filter(s -> !s.isEmpty()).collect(Collectors.toSet());
+      // Map all files/dirs to real location
+      arguments =
+          Arrays.stream(args)
+              .skip(7)
+              .map(arg -> arg.startsWith("-") ? arg : lookUpRunfileWithFallback(runfiles, arg))
+              .collect(Collectors.toList());
```

Comment:
`arguments` could be of type `Stream<String>` to prevent the conversion into `List` and later back to `Stream`.